src/controller/python/test/test_scripts/base.py

```diff
@@ -387,12 +365,12 @@ def TestFailsafe(self, nodeid: int):
         self.logger.info(
             "Attempting to open basic commissioning window - this should fail since the failsafe is armed")
         try:
-            asyncio.run(self.devCtrl.SendCommand(
+            await self.devCtrl.SendCommand(
                 nodeid,
                 0,
                 Clusters.AdministratorCommissioning.Commands.OpenBasicCommissioningWindow(180),
                 timedRequestTimeoutMs=10000
-            ))
+            )
             # we actually want the exception here because we want to see a failure, so return False here
             self.logger.error(
                 'Incorrectly succeeded in opening basic commissioning window')
```

Comment:
nit, I know that this is outside your PR but can you update this catch of all exceptions to catching the excepted exception (in code below)?

If that is too much maybe add a TODO with a link to github issue number

Reply:
And here too!